lessons/basics/modules.md

```diff
@@ -225,15 +254,58 @@ defmodule UseImportRequire.UseMe do
 end
 ```
 
-and we add this line to UseImportRequire:
+and we add this line to use UseImportRequire:
 
 ```elixir
 use UseImportRequire.UseMe
 ```
 
-Using UseImportRequire.UseMe defines a use_test/0 function through invocation of the `__using__/1` macro.
+Using `UseImportRequire.UseMe` defines a `use_test/0` function through invocation of the `__using__/1` macro.
+
+```elixir
+iex> UseImportRequire.use_test
+use_test
+:ok
+```
+
+We can also override a given function that is given to use with `__using__/1`.
```

Comment:
I'm not sure about this, for one there are too many "givens" but this seems kind of out-of-place, there's no lead-in or reason.  Maybe we can reword it or maybe it should come at the end of the section on `use`?

How about something like:

"Sometimes it may be necessary to allow certain functions within our macro to be overridden, to this we'll need to use `defoverridable`.

Let's look at how we can leverage this:"